src/components/Nav/Nav.js

```diff
@@ -74,6 +77,55 @@ function Nav() {
     return () => document.removeEventListener('mousedown', handleClickOutside)
   })
 
+  useEffect(() => {
+    if (router.pathname === '/') {
+      setMainContentLink('/#our_stories')
+    } else if (router.pathname === '/about') {
+      setMainContentLink('/about#about')
+    } else if (router.pathname === '/board') {
+      setMainContentLink('/board#about')
+    } else if (router.pathname === '/testimonials') {
+      setMainContentLink('/testimonials#about')
+    } else if (router.pathname === '/blog') {
+      setMainContentLink('/blog#content')
+    } else if (router.pathname === '/podcast') {
+      setMainContentLink('/podcast#content')
+    } else if (router.pathname === '/apply') {
+      setMainContentLink('/apply#contact')
+    } else if (router.pathname === '/mentor') {
+      setMainContentLink('/mentor#contact')
+    } else if (router.pathname === '/syllabus') {
+      setMainContentLink('/syllabus#contact')
+    } else if (router.pathname === '/jobs') {
+      setMainContentLink('/jobs#jobs')
+    } else if (router.pathname === '/contact') {
+      setMainContentLink('/contact#contact')
+    } else if (router.pathname === '/code-of-conduct') {
+      setMainContentLink('/code-of-conduct#about')
+    } else if (router.pathname === '/donate') {
+      setMainContentLink('/donate#cause_single')
+    }
+  }, [router.pathname])
+
+  useEffect(() => {
+    if (applyTabIndex) {
+      const elementId = mainContentLink.substring(
+        mainContentLink.length,
+        mainContentLink.indexOf('#') + 1
+      )
+      const skipToElement = document.getElementById(elementId)
+
+      skipToElement.setAttribute('tabIndex', 0)
```

Comment:
@rmrose78 Can you help me understand why we are manually setting tabIndex this is really a last resort way to fix focusing an element.

Reply:
@andrew-t-james The first reason was that the content Jerome wanted the skip nav to go to on the home page wasn't targetable unless tabIndex was used. Open to better solutions, but when I was looking it seemed that most solutions fell back on tabIndex. 

The second reason is that adding the tabIndex and then focusing it takes the user's focus target off the navigation so they can tab down the page. After focusing on it I use the removeAttribute to take the tabIndex off.